docs/tutorials.rst

```diff
@@ -250,41 +258,39 @@ your application.
 .. code-block:: python
    :linenos:
 
-    import random
-    import string
-
-    import cherrypy
-
-    class StringGenerator(object):
+   import random
+   import string
+   
+   import cherrypy
+   
+   
+   class StringGenerator(object):
        @cherrypy.expose
        def index(self):
            return """<html>
              <head></head>
-         <body>
-           <form method="get" action="generate">
-             <input type="text" value="8" name="length" />
+             <body>
+               <form method="get" action="generate">
+                 <input type="text" value="8" name="length" />
                  <button type="submit">Give it now!</button>
-           </form>
-         </body>
-       </html>"""
-
+               </form>
+             </body>
+           </html>"""
+   
        @cherrypy.expose
        def generate(self, length=8):
            some_string = ''.join(random.sample(string.hexdigits, int(length)))
            cherrypy.session['mystring'] = some_string
            return some_string
-
+   
        @cherrypy.expose
        def display(self):
            return cherrypy.session['mystring']
-
-    if __name__ == '__main__':
-        conf = {
-            '/': {
-                'tools.sessions.on': True
-            }
-        }
-        cherrypy.quickstart(StringGenerator(), '/', conf)
+   
+   
+   if __name__ == '__main__':
+       conf = {'/': {'tools.sessions.on': True}}
```

Comment:
I'm -1 on this change (compacting the config). It's common for CherryPy apps to write this conf as written because it's clearer to see which sections apply, but also because it's highly likely one will want to add more lines to the config, so using a format that's conducive to adding more lines without requiring reformatting is best. Let's reindent this one and others if they're similar.